shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -77,7 +77,9 @@ public void onMethodCall(MethodCall call, Result result) {
         }
     }
 
-    private static int inputTypeFromTextInputType(String inputType, boolean obscureText) {
+    private static int inputTypeFromTextInputType(String inputType,
+                                                  boolean obscureText,
+                                                  boolean noAutoCorrect) {
```

Comment:
I'd like false to represent the default behavior and true to represent the special case (similar to obscureText)